### PR TITLE
Feat/shell snippets

### PR DESCRIPTION
**Pull request by madrigal1**
adds scripts to quickly add snippets

### CODE REVIEW

**Comment by madrigal1** [comment]
Closes #10

File: README.md [hunk 1]
```diff
@@ -152,6 +152,54 @@ Nap is customized through environment variables:
   />
 </p>
 
+## Snippet Management from Terminal
+- Emulate the functionality of cmd snippet managers like like pet(https://github.com/knqyf263/pet) and the-way(https://github.com/out-of-cheese-error/the-way)
```
Comment (jolheiser) [inline README.md:156]:
```suggestion
- Emulate the functionality of cmd snippet managers like [pet](https://github.com/knqyf263/pet) and [the-way](https://github.com/out-of-cheese-error/the-way)
```

Comment (madrigal1) [reply README.md]: Hey thanks for the review @jolheiser. I have pushed the commit with the specified change as requested. Hope you find the changes good enough to be merged.Do let me know if any further changes are required

File: README.md [hunk 2]
```diff
@@ -152,6 +152,54 @@ Nap is customized through environment variables:
   />
 </p>
 
+## Snippet Management from Terminal
```
Comment (maaslalani) [inline README.md:155]: Hey, thanks for this PR, I think we can simplify a lot of these functions to simply be aliases:

```
alias napclip="nap $(nap list | gum filter) | xclip -selection clipboard"
alias napexec="nap $(nap list | gum filter) | bash"
alias napsave="gum write | nap $(gum input --placeholder folder/snippet.ext)"
```

**Comment by pr3c0g** [comment]
I'm very interested in this PR because it's the main way I use nap.
One of the things I would like to have is having snippets with comments that are visible while exploring/searching through them, but ignored when copied. Should I open a separate PR for this?

**Comment by jolheiser** [comment]
This PR is just docs, so you should be able to implement these ideas as-is.

> One of the things I would like to have is having snippets with comments that are visible while exploring/searching through them, but ignored when copied.

Yes, feel free to open another PR for this! 🙂 
Sounds like an interesting use-case.

**Comment by maaslalani** [comment]
Thanks so much @madrigal1! I will close this PR for now but I really appreciate you creating this PR!

**Comment by kugarocks** [comment]
> I'm very interested in this PR because it's the main way I use nap. One of the things I would like to have is having snippets with comments that are visible while exploring/searching through them, but ignored when copied. Should I open a separate PR for this?

I have make a fork version for comments. You can checkout [Markdown Finder](https://github.com/kugarocks/markdown-finder) which is a tool for quickly locating code snippets in Markdown files. Use --- to divide sections of a Markdown file, making it easy to copy code from each section. @pr3c0g @maaslalani

**Pull request closed without merging**